tests/gentropy/method/test_susie_inf.py

```diff
@@ -48,3 +52,28 @@ def test_SUSIE_inf_cred(
         )
         cred = SUSIE_inf.cred_inf(susie_output["PIP"], LD=ld)
         assert cred[0] == [5]
+
+    def test_SUSIE_inf_convert_to_study_locus(
+        self: TestSUSIE_inf,
+        sample_data_for_susie_inf: list[np.ndarray],
+        sample_summary_statistics: SummaryStatistics,
+        session: Session,
+    ) -> None:
+        """Test of SuSiE-inf credible set generator."""
+        ld = sample_data_for_susie_inf[0]
+        z = sample_data_for_susie_inf[1]
+        susie_output = SUSIE_inf.susie_inf(
+            z=z,
+            LD=ld,
+            est_tausq=False,
+        )
+        gwas_df = sample_summary_statistics._df.limit(21)
+        L1 = SusieFineMapperStep.susie_inf_to_studylocus(
+            susie_output=susie_output,
+            session=session,
+            _studyId="sample_id",
+            _region="sample_region",
+            _join=gwas_df,
+            cs_lbf_thr=2,
+        )
+        assert isinstance(L1, StudyLocus), "L1 is not an instance of StudyLocus"
```

Comment:
I'm not sure how does the test dataset look like, it would be great to assert that the number of credible set is what you are expecting, and validate if the `locus` object is healthy. However I understand if that is not a high priority for now.

Reply:
I think we can create more meaningful test for bigger function that will use this convertor on later stages.